CSS/css_completions.py

```diff
@@ -36,7 +36,44 @@
     'break_inside': [
         'auto', 'avoid', 'avoid-page', 'avoid-column', 'avoid-region'
     ],
-    'color': ['currentColor', 'rgb($1)', 'rgba($1)', 'hsl($1)', 'hsla($1)', 'transparent'],
+    'color': [
+        'rgb($1)', 'rgba($1)', 'hsl($1)', 'hsla($1)', 'hwb($1)',
+        'blend($1)', 'blenda($1)', 'color($1)', 'shade($1)', 'tint($1)',
```

Comment:
I believe these are part of the color module v4, which the last I checked was in fairly heavy flux. I know we implemented things like `blend()`, `blenda()`, etc in minihtml, but they've since decided to rip a whole bunch of that out since they wouldn't do what people expected since they defaulted to the RGB color space instead of Lab.